src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -806,72 +687,173 @@ export default class TaskRoleContainerList extends React.Component {
           </div>
         ),
       },
+      exitTypeColumn,
+      {
+        key: 'exitCode',
+        name: 'Exit Code',
+        minWidth: 260,
+        headerClassName: FontClassNames.medium,
+        isResizable: true,
+        onRender: item => {
+          return isNil(item.containerExitSpec) ? (
+            <div className={c(FontClassNames.mediumPlus)}>
+              {item.containerExitCode}
+            </div>
+          ) : (
+            <div className={c(FontClassNames.mediumPlus)}>
+              {`${item.containerExitCode} (${item.containerExitSpec.phrase})`}
+            </div>
+          );
+        },
+      },
+    ];
+    const optionalColumns = [
+      {
+        key: 'createdTime',
+        name: 'Created Time',
+        headerClassName: FontClassNames.medium,
+        minWidth: 180,
+        maxWidth: 200,
+        isResizable: true,
+        onRender: item => {
+          return (
+            <div className={c(FontClassNames.mediumPlus)}>
+              {isNil(item.createdTime)
+                ? 'N/A'
+                : DateTime.fromMillis(item.createdTime).toLocaleString(
+                    DateTime.DATETIME_MED_WITH_SECONDS,
+                  )}
+            </div>
+          );
+        },
+      },
+      {
+        key: 'Duration',
+        name: 'Duration',
```

Comment:
>Duration [](start = 15, length = 8)

As Task createdTime nearly the same as job attempt created time, so change to "Running Duration" which can be got from this.getTimeDuration(item.launchedTime, item.completedTime) ?

Reply:
done